src/main/java/io/openliberty/tools/common/plugins/util/AbstractContainerSupportUtil.java

```diff
@@ -48,22 +132,26 @@ protected String execDockerCmd(String command, int timeout, boolean throwExcepti
             }
             result = readStdOut(p);
         } catch (IllegalThreadStateException  e) {
-            // the timeout was too short and the docker command has not yet completed. There is no exit value.
+            // the timeout was too short and the container command has not yet completed. There is no exit value.
             debug("IllegalThreadStateException, message="+e.getMessage());
-            error("The docker command did not complete within the timeout period: " + timeout + " seconds.", e);
-            throw new RuntimeException("The docker command did not complete within the timeout period: " + timeout + " seconds. ");
+            error("The container command did not complete within the timeout period: " + timeout + " seconds.", e);
+            throw new RuntimeException("The container command did not complete within the timeout period: " + timeout + " seconds. ");
         } catch (InterruptedException e) {
             // If a runtime exception occurred in the server task, log and rethrow
-            error("An interruption error occurred while running a docker command: " + e.getMessage(), e);
+            error("An interruption error occurred while running a container command: " + e.getMessage(), e);
             throw new RuntimeException(e.getMessage());
         } catch (IOException e) {
-            // If a runtime exception occurred in the server task, log and rethrow
-            error("An error occurred while running a docker command: " + e.getMessage(), e);
-            throw new RuntimeException(e.getMessage());
+            // Logging IOExceptions in info stream. This is thrown if Docker or Podman are not installed on the system.
+            info("An error occurred while running a container command: " + e.getMessage());
+            info("This message will occur when Docker or Podman are not installed.");
         }
         return result;
     }
 
+    protected String execContainerCmd(String command, int timeout) {
+        return execContainerCmd(command, timeout, false);
```

Comment:
Do you really want the boolean to be `false` here? If so, I think the method name should be different...maybe `execContainerCmdSuppressError`.